api/lambda_function.py

```diff
@@ -11,8 +11,17 @@ def lambda_handler(event, context):
     request_type = event['requestType']
     store_id = event['storeId']
     customer_id = event['customerId']
-
-    response = {}
+    # write store_id & customer_id to the DynamoDB table using the object we instantiated and save response in a variable
+    response = table.put_item(
+        Item={
+            'storeId': store_id
```

Comment:
The database schema is StoreId, CustomerId, Stamps. So storeId should be capitalized to StoreId and so on